packages/web3-eth-contract/src/contract.ts

```diff
@@ -126,19 +141,84 @@ export class Contract<Abi extends ContractAbi>
 	>
 	implements Web3EventEmitter<ContractEventEmitterInterface<Abi>>
 {
+	/**
+	 * The options `object` for the contract instance. `from`, `gas` and `gasPrice` are used as fallback values when sending transactions.
+	 *
+	 * ```ts
+	 * myContract.options;
+	 * > {
+	 *     address: '0x1234567890123456789012345678901234567891',
+	 *     jsonInterface: [...],
+	 *     from: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe',
+	 *     gasPrice: '10000000000000',
+	 *     gas: 1000000
+	 * }
+	 *
+	 * myContract.options.from = '0x1234567890123456789012345678901234567891'; // default from address
+	 * myContract.options.gasPrice = '20000000000000'; // default gas price in wei
+	 * myContract.options.gas = 5000000; // provide as fallback always 5M gas
+	 * ```
+	 */
 	public readonly options: ContractOptions;
 
+	/**
+	 * Can be used to set {@link Contract.defaultAccount} for all contracts.
+	 */
 	public static defaultAccount?: HexString;
+
+	/**
+	 * Can be used to set {@link Contract.defaultBlock} for all contracts.
```

Comment:
Same for default block:
You can override it by passing in the defaultBlock as last parameter. The default value is "latest".
The default block parameters can be one of the following:

blocknumber
"earliest" - String: The genesis block
"latest" - String: The latest block (current head of the blockchain)
"pending" - String: The currently mined block (including pending transactions)
Default is "latest".


Same for all other default fields, we should add more details.

Reply:
As commented above, all these documentation is in the scope of the `Web3Context` not the `contracts` package and shared among all other packages. 

It's referenced for the `Contract` because it's inherits these attributes from the context.